flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
   languages?: Array<string>,
 };
-
```

Comment:
:small_red_triangle:  Playlist items not resolving right
- http://localhost:9090/@TheRingOfFire:c/book-claims-giuliani-came-up-with:d?lid=8a4b0e61bfb8ba64203d44aacfefe2964c3b875a  this should be loading 4 items (compare with master)

The playlist is from Tom's channel. Just keep refreshing and/or navigating back and forth between the various view of a playlist.  I think it's not resolving right:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/64950861/189707899-47b0183d-b685-4f39-a715-6b3fda6e4683.png">

Reply:
Even when it resolves, the last 2 items are duplicates, as warned by the console.  They are not in master.

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/64950861/189708198-785eb766-3dd2-4343-8054-ccb501617bc2.png">

Reply:
Reproducible, but no exact recipe. Seems to involve Play or Shuffle Play

<img width="1222" alt="image" src="https://user-images.githubusercontent.com/64950861/189709636-8ee6bcce-b8d9-4dd4-974b-846083830c39.png">

Reply:
Not sure what triggers it, but feels like the claim is being deleted or the selector filtered it out?

https://user-images.githubusercontent.com/64950861/189809792-233b43d6-ff8c-452b-bbc5-74d336bd2124.mp4